app/models/manageiq/providers/openstack/network_manager/floating_ip.rb

```diff
@@ -42,6 +41,12 @@ def self.remapping(options)
     new_options[:floating_ip_address] = options[:address] if options[:address]
```

Comment:
Hi Sam, sure, sorry for later reponse. Looks good, but from my point of view it would be even better adding ```.present?``` to the end of this line instead of allow setting option to nil or empty and deleting it later.

```new_options[:floating_ip_address] = options[:address] if options[:address].present?```

Reply:
The UI form actually sets `new_options[:floating_ip_address]`, so that does need to be tested and blank values deleted either way. I'm not sure what the purpose is of the `options[:address]` line, but I've left it there since I assume it's there for a reason.